File: VB/WebSite/UserForms/ScriptRecurrenceForm.ascx.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Data
Imports System.Configuration
Imports System.Collections
Imports System.Web
Imports System.Web.Security
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports System.Web.UI.HtmlControls
Imports DevExpress.Web.ASPxScheduler

Partial Public Class UserForms_ScriptRecurrenceForm
	Inherits ASPxSchedulerClientFormBase
	Public Overrides ReadOnly Property ClassName() As String
		Get
			Return "ASPxClientRecurrenceAppointmentForm"
		End Get
	End Property

	Protected Overrides Function GetChildControls() As Control()
		Dim controls() As Control = { edtDailyRecurrenceControl, edtWeeklyRecurrenceControl, edtMonthlyRecurrenceControl, edtYearlyRecurrenceControl, edtRecurrenceTypeEdit, edtRecurrenceRangeControl}
		Return controls
	End Function
End Class

File: CS/WebSite/UserForms/ScriptRecurrenceForm.ascx.cs
using System;
using System.Data;
using System.Configuration;
using System.Collections;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;
using DevExpress.Web.ASPxScheduler;

public partial class UserForms_ScriptRecurrenceForm : ASPxSchedulerClientFormBase {
    public override string ClassName { get { return "ASPxClientRecurrenceAppointmentForm"; } }

    protected override Control[] GetChildControls() {
        Control[] controls = new Control[] { edtDailyRecurrenceControl, edtWeeklyRecurrenceControl, 
            edtMonthlyRecurrenceControl, edtYearlyRecurrenceControl, 
            edtRecurrenceTypeEdit, edtRecurrenceRangeControl};
        return controls;
    }
}
